ecommerce/views.py

```diff
@@ -202,3 +205,27 @@ def get(self, request, username, *args, **kwargs):  # pylint: disable=unused-arg
         )
 
         return Response(serialize_user_bootcamp_runs(user=user))
+
+
+class CheckoutDataViewSet(ReadOnlyModelViewSet):
+    """
+    List application ecommerce data for a user, for payable applications
+    """
+    authentication_classes = (
+        SessionAuthentication,
+    )
+    permission_classes = (
+        IsAuthenticated,
+    )
+    serializer_class = CheckoutDataSerializer
+
+    def get_queryset(self):
```

Comment:
We should select/prefetch tables that we know we're going to need. Bootcamp run, installments, and orders/lines come to mind